.gitignore

```diff
@@ -0,0 +1,9 @@
+.idea/
+.vscode/
+*.iml
+.env
+.DS_Store
+venv/
+.pytest_cache/
+**__pycache__/
+db.sqlite3
```

Comment:
add a blank line at the end of the file.

Reply:
Done